SampleCode/Cxx11SGXDemo/App/App.cpp

```diff
@@ -98,8 +98,8 @@ static sgx_errlist_t sgx_errlist[] = {
         "Please make sure SGX module is enabled in the BIOS, and install SGX driver afterwards."
     },
     {
-        SGX_ERROR_MEMORY_MAP_CONFLICT,
-        "Memory map conflicted.",
+        SGX_ERROR_MEMORY_MAP_FAILURE,
+        "Failed to reserve memory for the enclave.",
```

Comment:
I would prefer to add one more description for SGX_ERROR_MEMORY_MAP_FAILURE here, not to replace the description for SGX_ERROR_MEMORY_MAP_CONFLICT

Reply:
It's a good plan if both SGX_ERROR_MEMORY_MAP_CONFLICT and SGX_ERROR_MEMORY_MAP_FAILURE will return. But actually only SGX_ERROR_MEMORY_MAP_FAILURE will return even if memory map conflict.

Reply:
There could be a small possibility that someone use the old urts and old driver to retry the app, so SGX_ERROR_MEMORY_MAP_CONFLICT could be returned in some corner case.
So we prefer to add the description here than replace the description.

Reply:
ok, i kown it.
In this case the SDK retry mechanism should judment the return val both SGX_ERROR_MEMORY_MAP_CONFLICT and SGX_ERROR_MEMORY_MAP_CONFLICT to compatible with older driver. Whether new SDK should use new driver and new urts.

Reply:
yeah, keep it in the example files just in case. We can remove it when we are sure it is not needed

Reply:
ok, i fix my code. please check it.
one question, Is the SGX_ERROR_MEMORY_MAP_CONFLICT an error type of SGX_ERROR_MEMORY_MAP_FAILURE?

Reply:
No, they are different error codes.